static-assets/js/custom.js

```diff
@@ -31,10 +32,29 @@ $(function(){
         $(".navbar-collapse").collapse('hide');
     });
 
-    // NIVO LIGHTBOX
-    $('#gallery a').nivoLightbox({
-        effect: 'fadeScale',
-    });
+    // LIGHTBOX
+    let lightbox;
+    const initLightbox = () => {
+        lightbox = new SimpleLightbox({
+            $items: $('#gallery a')
+        });
+    }
+
+    if (isAuthoring) {
+        let isEditMode;
+        document.addEventListener('craftercms.editMode', (e) => {
+            isEditMode = e.detail;
+            if (isEditMode) {
+                lightbox.destroy();
+            } else {
+                initLightbox();
+            }
```

Comment:
This needs to run always when the page is loaded. The event might be missed the first time. Check the edit mode class to run the first initialization.

Reply:
If when checking the edit mode class XB hasn't been initialized, then the lightbox would be incorrectly initialized (since the class would not be set yet). Do we have a way of waiting until xb has been initialized so we can check for those values?